doc/SUMMARY.md

```diff
@@ -9,9 +9,11 @@
 * [Services and Contributions](Services_and_Contributions.md)
 
 ## Using Theia
+
 * [Build your own IDE](Composing_Applications.md)
 * [Authoring an Extension](Authoring_Extensions.md)
 * [Authoring Plug-ins](Authoring_Plugins.md)
+* [TextMate Coloring](TextMate.md)
```

Comment:
TextMate Syntax Coloring?

Reply:
I think it should be 'Adding Language Support'. 
And then 'the grammar (textmate) would be one part.
Configuration and language server two others.